engine/src/main/resources/assets/shaders/tint_vert.glsl

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2012 Benjamin Glatzel <benjamin.glatzel@me.com>
```

Comment:
What's this going to be used for?

Reply:
mistakenly copypasted copyright, shader should've been moved to module repo, will remove file